initializr-service/src/main/resources/application.yml

```diff
@@ -91,6 +91,28 @@ initializr:
     kotlin:
       version: 1.1.2-5
   dependencies:
+    - name: Azure
+      content:
+        - name: Azure Storage
+          id: azure-storage
+          groupId: com.microsoft.azure
+          artifactId: azure-storage-spring-boot-starter
+          description: Spring Boot starter for Azure Storage service
+          version: 0.1.0
```

Comment:
The version must be upgraded somewhere. Usually we require a bom because that makes version management more explicit and easy to use. You can look at that file for other artifacts and you can see there is a range mapping for Spring Boot generations (look at the `spring-cloud` bom for instance).

Let's first decide the outcome of this PR and we can discuss this in more details if necessary.